awsshell/compat.py

```diff
@@ -10,11 +10,13 @@
 if PY3:
     from html.parser import HTMLParser
     text_type = str
+    long_type = int
     from io import StringIO
     import dbm
 else:
     from HTMLParser import HTMLParser
     text_type = unicode
+    long_type = long
```

Comment:
I don't think so. The six.integer_types is `(int, long)` in python 2 and just `int` in python 3. This makes it not possible to use the value as a constructor to coerce the parameter. That being said after doing some research it seems that calling `int` on a large enough number will end up returning a `long` anyways and this may not be needed.